bulkimport_example_test.go

```diff
@@ -0,0 +1,115 @@
+package mssql_test
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"unicode/utf8"
+
+	mssql "github.com/denisenkom/go-mssqldb"
+)
+
+const (
+	createTestTable = `CREATE TABLE test_table(

```

Comment:
I have a personal preference in SQL to not quote column names and types when not needed, but I generally don't use a tool to generate sql or schemas.